src/axom/quest/MarchingCubes.hpp

```diff
@@ -0,0 +1,295 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/*!
+ * \file MarchingCubes.hpp
+ *
+ * \brief Consists of class implementing marching cubes algorithm to
+ * compute iso-surface from a scalar field in a blueprint mesh.
+ */
+
+#ifndef AXOM_PRIMAL_MARCHINGCUBES_H_
+#define AXOM_PRIMAL_MARCHINGCUBES_H_
+
+// Axom includes
+#include "axom/config.hpp"
+#include "axom/mint/mesh/Mesh.hpp"
+
+// Conduit includes
+#include "conduit_node.hpp"
+
+// C++ includes
+#include <string>
+
+namespace axom
+{
+namespace quest
+{
+/*!
+ * \@brief Class implementing marching cubes algorithm on a single
+ * structured mesh domain within a multi-domain mesh.
+ *
+ * Implementation is for 2D (marching squares) and 3D (marching
+ * cubes).
+ *
+ * The input mesh is a Conduit::Node following the Mesh Blueprint
+ * convention.
+ *
+ * Usage example:
+ * @beginverbatim
+ *   MarchingCubes computationalMesh(meshNode, "coords");
+ *   axom::mint::UnstructuredMesh<axom::mint::_SINGLe_SHAPE>
+ *     surfaceMesh(3, min::CellType::Triangle);
```

Comment:
```suggestion
 *   axom::mint::UnstructuredMesh<axom::mint::SINGLE_SHAPE>
 *     surfaceMesh(3, min::CellType::Triangle);
```

I think this is a typo.